docs/source/simulation-guide/simulation-types.md

```diff
@@ -115,4 +119,56 @@ no information in IFC we can either request them during export or mark them as
 non-existing in the exported model, so the user can input them in Modelica.
 2. For custom controls and complex controls which highly depend on the system
 and the usage we offer the possibility to deactivate the internal controls inside
-the modelica models and allow the user to model their own controls.
\ No newline at end of file
+the modelica models and allow the user to model their own controls.
+
+(simtype_cfd)=
+## Computational Fluid Dynamics (CFD) Simulations
+
+CFD simulations are used to analyze fluid flow, heat transfer, and indoor air 
+distribution in buildings. While Building Performance Simulation (BPS) focuses 
+on overall energy balance and system sizing, CFD provides detailed insights 
+into local comfort parameters, such as air velocity, temperature stratification, 
+and contaminant dispersion. This allows designers to assess the indoor 
+environmental quality (IEQ) beyond energy efficiency.
```

Comment:
Please adjust similar to my previous comment. Add something on single-node-per-zone BPS and high-resolution spatial grids for CFD.